packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -40,22 +39,20 @@ function Loading({ show }: { show?: boolean }) {
   return show ? <LoadingControl /> : null;
 }
 
-export const LocationDetailMessage = (): React.JSX.Element | null => {
-  const [{ hasError, message }] = useAction('LIST_LOCATION_ITEMS');
-
-  return hasError ? (
+export const LocationDetailMessage = ({
+  show,
+  message,
+}: {
+  show?: boolean;
+  message?: string;
```

Comment:
Don't need `show`, can gate rendering based on the presence of `message`

Reply:
I thought about that but we have a `DEFAULT_ERROR_MESSAGE `